### PR TITLE
[SMALLFIX][branch-1.6] Add missing configuration properties

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: core/common/src/main/java/alluxio/PropertyKey.java [hunk 1]
```diff
@@ -316,20 +321,24 @@ public String toString() {
...
   public static final PropertyKey UNDERFS_OSS_CONNECT_TIMEOUT =
       new Builder(Name.UNDERFS_OSS_CONNECT_TIMEOUT)
           .setAlias(new String[]{"alluxio.underfs.oss.connection.timeout.ms"})
           .setDefaultValue("50sec")
+          .setDescription("The timeout when connecting to OSS")
           .build();
   public static final PropertyKey UNDERFS_OSS_CONNECT_TTL =
       new Builder(Name.UNDERFS_OSS_CONNECT_TTL)
           .setDefaultValue(-1)
+          .setDescription("The TTL of OSS connections in ms")
```
Comment (gpang) [inline core/common/src/main/java/alluxio/PropertyKey.java]: Do you need the "in ms" part?

File: core/common/src/main/java/alluxio/PropertyKey.java [hunk 2]
```diff
@@ -823,7 +864,8 @@ public String toString() {
               + "folder for Alluxio worker to put data for tiered store.")
           .build();
   public static final PropertyKey WORKER_DATA_HOSTNAME =
-      new Builder(Name.WORKER_DATA_HOSTNAME).build();
+      new Builder(Name.WORKER_DATA_HOSTNAME)
+          .setDescription("The hostname of Alluxio data service").build();
```
Comment (gpang) [inline core/common/src/main/java/alluxio/PropertyKey.java]: "Alluxio worker"?

File: core/common/src/main/java/alluxio/PropertyKey.java [hunk 3]
```diff
@@ -896,13 +938,13 @@ public String toString() {
   public static final PropertyKey WORKER_FILE_BUFFER_SIZE =
       new Builder(Name.WORKER_FILE_BUFFER_SIZE)
           .setDefaultValue("1MB")
+          .setDescription("When worker writing data, the size of initial buffer")
```
Comment (gpang) [inline core/common/src/main/java/alluxio/PropertyKey.java]: The size of the initial buffer, when the worker writes data.

Is this when the worker is writing to a block? to a ufs file?

File: core/common/src/main/java/alluxio/PropertyKey.java [hunk 4]
```diff
@@ -1194,16 +1266,23 @@ public String toString() {
       new Builder(Name.PROXY_STREAM_CACHE_TIMEOUT_MS)
           .setAlias(new String[]{"alluxio.proxy.stream.cache.timeout.ms"})
           .setDefaultValue("1hour")
+          .setDescription("The timeout for the input and output streams cache eviction in the "
+              + "proxy..")
```
Comment (gpang) [inline core/common/src/main/java/alluxio/PropertyKey.java]: double ".."

File: core/common/src/main/java/alluxio/PropertyKey.java (hunk 1 above)
Comment (apc999) [reply core/common/src/main/java/alluxio/PropertyKey.java]: yes, this is actually an OSS property , which can take -1. If we use `getMs`. I am not sure if it takes -1. so I didn't change it to use `getMs` but leave it as what OSS client requires

File: core/common/src/main/java/alluxio/PropertyKey.java [hunk 5]
```diff
@@ -1489,28 +1559,32 @@ public String toString() {
   public static final PropertyKey USER_NETWORK_NETTY_WRITER_PACKET_SIZE_BYTES =
       new Builder(Name.USER_NETWORK_NETTY_WRITER_PACKET_SIZE_BYTES)
           .setDefaultValue("64KB")
+          .setDescription("When a client writing to a worker through network, the maximum packet "
```
Comment (aaudiber) [inline core/common/src/main/java/alluxio/PropertyKey.java]: writing -> writes

through network -> over the network

File: core/common/src/main/java/alluxio/PropertyKey.java [hunk 6]
```diff
@@ -1489,28 +1559,32 @@ public String toString() {
...
+              + "size.")
           .build();
   public static final PropertyKey USER_NETWORK_NETTY_WRITER_BUFFER_SIZE_PACKETS =
       new Builder(Name.USER_NETWORK_NETTY_WRITER_BUFFER_SIZE_PACKETS)
           .setDefaultValue(16)
+          .setDescription("When a client writing to a worker through network, the buffer size in "
```
Comment (aaudiber) [inline core/common/src/main/java/alluxio/PropertyKey.java]: writing -> writes

Comment (aaudiber) [reply core/common/src/main/java/alluxio/PropertyKey.java]: through network -> over the network

Comment (aaudiber) [inline core/common/src/main/java/alluxio/PropertyKey.java]: the maximum number of packets to buffer

Comment (aaudiber) [inline core/common/src/main/java/alluxio/PropertyKey.java]: specify that the buffering happens on the client

File: core/common/src/main/java/alluxio/PropertyKey.java [hunk 7]
```diff
@@ -1489,28 +1559,32 @@ public String toString() {
...
+          .setDescription("The timeout to close a netty writer client.")
           .build();
   public static final PropertyKey USER_NETWORK_NETTY_READER_BUFFER_SIZE_PACKETS =
       new Builder(Name.USER_NETWORK_NETTY_READER_BUFFER_SIZE_PACKETS)
           .setDefaultValue(16)
-          .build();
-  public static final PropertyKey USER_NETWORK_NETTY_READER_CANCEL_ENABLED =
-      new Builder(Name.USER_NETWORK_NETTY_READER_CANCEL_ENABLED)
-          .setDefaultValue(true)
+          .setDescription("When a client reading from a worker through network, the buffer size in "
```
Comment (aaudiber) [inline core/common/src/main/java/alluxio/PropertyKey.java]: reading -> reads

Comment (aaudiber) [inline core/common/src/main/java/alluxio/PropertyKey.java]: through network -> over the network

File: core/common/src/main/java/alluxio/PropertyKey.java [hunk 8]
```diff
@@ -1489,28 +1559,32 @@ public String toString() {
...
+              + "terms of packets.")
           .build();
   public static final PropertyKey USER_NETWORK_NETTY_READER_PACKET_SIZE_BYTES =
       new Builder(Name.USER_NETWORK_NETTY_READER_PACKET_SIZE_BYTES)
           .setDefaultValue("64KB")
+          .setDescription("When a client reading from a worker through network, the maximum packet "
```
Comment (aaudiber) [inline core/common/src/main/java/alluxio/PropertyKey.java]: reads from a worker over the network

File: core/common/src/main/java/alluxio/PropertyKey.java [hunk 9]
```diff
@@ -1618,37 +1683,36 @@ public String toString() {
...
           .setDefaultValue("/")
+          .setDescription("The Alluxio path mounted to the FUSE file system root")
           .build();
-  /**
-   * Passed to fuse-mount, maximum granularity of write operations:
-   * Capped by the kernel to 128KB max (as of Linux 3.16.0),.
-   */
   public static final PropertyKey FUSE_MAXWRITE_BYTES =
       new Builder(Name.FUSE_MAXWRITE_BYTES)
           .setDefaultValue(131072)
```
Comment (aaudiber) [inline core/common/src/main/java/alluxio/PropertyKey.java]: "128KB"

File: core/common/src/main/java/alluxio/PropertyKey.java (hunk 2 above)
Comment (apc999) [reply core/common/src/main/java/alluxio/PropertyKey.java]: no it is only for dataserver

File: core/common/src/main/java/alluxio/PropertyKey.java (hunk 3 above)
Comment (apc999) [reply core/common/src/main/java/alluxio/PropertyKey.java]: fixed

File: core/common/src/main/java/alluxio/PropertyKey.java (hunk 4 above)
Comment (apc999) [reply core/common/src/main/java/alluxio/PropertyKey.java]: fixed

File: core/common/src/main/java/alluxio/PropertyKey.java (hunk 2 above)
Comment (aaudiber) [reply core/common/src/main/java/alluxio/PropertyKey.java]: "Alluxio worker data service"

**Comment by apc999** [comment]
@gpang @aaudiber comments addressed. PTAL

File: core/common/src/main/java/alluxio/PropertyKey.java (hunk 2 above)
Comment (apc999) [reply core/common/src/main/java/alluxio/PropertyKey.java]: fixed

**Pull request merged**